packages/ffe-file-upload/less/file-upload.less

```diff
@@ -0,0 +1,307 @@
+@ffe-blue-royal-escape: escape('@{ffe-blue-royal}');
+@ffe-blue-royal-fill: @ffe-blue-royal-escape;
+@ffe-blue-azure-escape: escape('@{ffe-blue-azure}');
+@ffe-blue-azure-fill: @ffe-blue-azure-escape;
+
+.ffe-file-upload {
+    margin-top: 45px;
+    display: grid;
+    max-width: 400px;
+    grid-template-columns: 1fr;
+
+    @media (min-width: @breakpoint-md) {
+        max-width: 100%;
+    }
+
+    &__title {
+        &:extend(.ffe-h5);
+
+        margin-bottom: 0;
+    }
+
+    input[type='file'] {
+        display: none;
+    }
+
+    &__info-section {
+        margin-bottom: 20px;
+        grid-row: 2;
+
+        @media (min-width: @breakpoint-md) {
+            margin-right: 20px;
+            margin-bottom: 0;
+            grid-column: 1;
+        }
+
+        &-text {
+            margin-bottom: 20px;
+        }
+    }
+
+    &__file-items-section {
+        margin-top: 20px;
+        line-height: 35px;
+        grid-row: 4;
+
+        @media (min-width: @breakpoint-md) {
+            margin-top: 0;
+            grid-row: 2;
+        }
+
+        li {
+            display: grid;
+            grid-template-columns: 1fr;
+        }
+    }
+
+    &__upload-section {
+        height: 150px;
+        background-color: @ffe-blue-pale;
+        border-radius: 10px;
+        text-align: center;
+        padding: 5px;
+        grid-row: 3;
+
+        @media (min-width: @breakpoint-md) {
+            margin-top: 0;
+            height: 220px;
+            width: 280px;
+            padding: 0;
+            grid-row: 2;
+            grid-column: 2;
+        }
+
+        &-border {
+            height: 140px;
+            margin: 5px;
+            border: 2px dashed @ffe-blue-azure;
+            border-radius: 10px;
+            position: relative;
+            width: 100%;
+            right: 5px;
+            bottom: 5px;
+
+            @media (min-width: @breakpoint-md) {
+                width: 270px;
+                height: 210px;
+                right: 0;
+                bottom: 0;
+            }
+
+            &__hover {
```

Comment:
this should be a modifier, i.e. `--hover`

Reply:
And out of curiosity, does the `:hover` pseudo selector not work here? Something to do with drag and drop?

Reply:
> this should be a modifier, i.e. `--hover`

It is a modifier, eg: 'block__component__modifier'

This modifier is only triggered when dragging files over the component to trigger the modified style, and as such I had to make a custom modifier. Other components, eg the 'delete' button, triggers on mouse-over ( the :hover selector ).

Reply:
> It is a modifier, eg: 'block__component__modifier'

It should be `block__element--modifier` as described by [BEM](http://getbem.com/naming/).
Thanks for clearing up why `:hover` isn't suited here.

Reply:
> > It is a modifier, eg: 'block__component__modifier'
> 
> It should be `block__element--modifier` as described by [BEM](http://getbem.com/naming/).
> Thanks for clearing up why `:hover` isn't suited here.

Ah, right! My bad :)